lib/engines/hcktest/hcktest.rb

```diff
@@ -254,6 +292,10 @@ def run
         dump_run
 
         @project.logger.info("Find all scripts in folder: #{@project.workspace_path}")
+        @project.logger.warn(
+          'Dump mode is only for basic tests. ' \
+          'For specific configurations (e.g., Secure Boot tests), update run scripts manually'
+        )
```

Comment:
Split this into two `warn` calls; if the message is too long in source code, it is probably too long to fit in the terminal as well.